components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -154,7 +165,7 @@ func (b *Beb) DeleteSubscription(subscription *eventingv1alpha1.Subscription) er
 func (b *Beb) deleteCreateAndHashSubscription(subscription *types.Subscription, cleaner eventtype.Cleaner, log *zap.SugaredLogger) (*types.Subscription, int64, error) {
 	// delete EMS subscription
 	if err := b.deleteSubscription(subscription.Name); err != nil {
-		log.Errorw("delete BEB subscription failed", "error", err)
+		log.Errorw("delete BEB subscription failed", "bebSubscriptionName", subscription.Name, "error", err)
```

Comment:
instead of adding bebSubscription name everywhere lets change the log object to include this key, `bebSubscriptionName `

Reply:
here is how the log is looking now:

{"level":"ERROR","timestamp":"2021-07-30T07:46:04Z","logger":"beb-handler","caller":"handlers/beb.go:121","message":"get BEB subscription failed","context":{"kind":"Subscription","version":1,"namespace":"test","name":"test","bebSubscriptionName":"test","error":"get subscription failed: 404; 404 Not Found"}}
{"level":"INFO","timestamp":"2021-07-30T07:46:04Z","logger":"beb-handler","caller":"handlers/beb.go:124","message":"Recreate the BEB subscription","context":{"kind":"Subscription","version":1,"namespace":"test","name":"test","bebSubscriptionName":"test"}}

We have, dependent of the calling point two subscriptions name:
- one for Kyma subscription: this is
```
"name":"test",
```
and one for beb subscription:
```
"bebSubscriptionName":"test"
```
I've put this, cause with the next PR, the two will be different and it's important to have in the log file the beb subscription name too.

Reply:
Not in all places used to log messages, the beb subscription exists, but in all places the kyma subscription exists.
This is why the log object contains only the kyma subscription and the beb subscription name is added as an additional parameter were it is available.